front/pages/api/w/[wId]/assistant/conversations/index.ts

```diff
@@ -86,6 +89,33 @@ async function handler(
       };
 
       if (contentFragments.length > 0) {
+        // When we send the attachments at the conversation creation, we are missing the useCaseMetadata
+        // Therefor, we couldn't upsert them to the conversation datasource.
+        // We now update the useCaseMetadata and upsert them to the conversation datasource.
+        const filesIds = removeNulls(
```

Comment:
Can we have a helper for this so we don't duplicate this logic in both endpoints?